src/scripts.js

```diff
@@ -109,11 +155,16 @@ function displayTravelerSpending(data) {
   domUpdates.updateTravelerSpending(data, yearlyCost);
 };
 
+
 function displayTravelerData(data) {
   domUpdates.updateWelcomeMessage(data);
   domUpdates.updateTravelerProfile(data);
 };
 
+function displayNumDaysTraveled(data) {
+  const daysTraveled = data.currentTraveler.findNumDaysTraveledThisYear();
+}
+
```

Comment:
If you're already aware of this, disregard, but I don't think this is hooked up to your sidebar yet!